com.unity.multiplayer.mlapi/Tests/Runtime/ConnectionApproval.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Text;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+
+namespace MLAPI.RuntimeTests
+{
+
+    public class ConnectionApprovalTests
+    {
+
+        private Guid m_ValidationToken;
+
+        private bool m_IsValidated;
+
+        [SetUp]
+        public void Setup()
+        {
+            // Create, instantiate, and host
+            Assert.IsTrue(NetworkManagerHelper.StartNetworkManager(out _,NetworkManagerHelper.NetworkManagerOperatingMode.None));
```

Comment:
@NoelStephensUnity @TwoTenPvP I like these helpers to simplify unit tests, but I think we may now have 2 sets of helpers.  Is this right?  Can we merge them into one?

Reply:
That is a good idea.